manifests/mariadb.pp

```diff
@@ -95,18 +86,18 @@
       $default_override_options,
       $ldap_override_options,
       $ssl_override_options,
-      $cluster_override_options,
-      ),
+      $cluster_override_options
+      )
   }
 
   class { '::mysql::client':
-    package_name => 'mariadb-client',
+    package_name => 'mariadb-client'
   }
 
   if ! empty($backup) {
-    class { '::vision_mysql::server::backup::client':
+    class { '::vision_mysql::server::backup':
       password  => $backup['password'],
-      databases => $backup['databases'],
+      databases => $backup['databases']
     }
   }
 }
```

Comment:
Ich bin ja eigentlich eher ein Freund der trailing commas in Puppet.
Wenn man eine neue Zeile hinzufuegt, muss man dann naemlich die darueberliegende nicht aendern (auch ganz nett fuer Git).

Reply:
:thinking:  I see... Yo das macht Sinn